sdk/servicebus/service-bus/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Release History
 
+## 2.0.0-preview.1 (#ToBeReleased)
+
+- Fixes [bug 6816](https://github.com/Azure/azure-sdk-for-js/issues/6816) affecting messages sent using the `scheduleMessage()` and `scheduleMessages()` methods. [PR 7372](https://github.com/Azure/azure-sdk-for-js/pull/7372)
```

Comment:
I noticed that we considered this a breaking change in the track1 library so only did this in track2. Should we add a 
`### Breaking Change` header and explain what the difference is for customers migrating from track1?
(Looking at https://azure.github.io/azure-sdk/policies_releases.html for an example of this)

Reply:
Added the breaking-change header and some description mentioning the difference and what would change from the user's perspective.
Please review.